### ISSUE TITLE
Ask users how much alcohol they consume

### ISSUE THREAD

**Issue by michaelNetCompany** [nhsuk/nhsuk-service-manual-community-backlog#545]
## What

Any services that ask health questions that require patients answering questions about their alcohol consumption.

## Why

This may become a common online question. It would be good to share and come up with a common pattern here - if we can.

## Other information to support your proposal

This is the current journey being used in the GP Registration service

![image](https://github.com/user-attachments/assets/c0f5ecfd-5bf9-4939-9883-e5d764c502e2)
![image](https://github.com/user-attachments/assets/7ce646b8-1d0f-4f65-b775-00fb0d34d009)
![image](https://github.com/user-attachments/assets/a84a927a-66f3-4a09-b2fe-2205c705d76a)

## Research
We've had a few instances of patient feedback that highlight that working out how many units of alcohol someone consumes is a difficult math equation

**Comment by sarawilcox** [comment]
We can't work out units for them, can we?

**Comment by michaelNetCompany** [comment]
> We can't work out units for them, can we?

I don't think we can work it out for them. We could potentially offer them broader categories to choose from, rather than provide an unlimited input box.

It all depends on what is done with the data once it's been collected.
